.github/workflows/docker-stack-sipnet.yml

```diff
@@ -62,6 +62,10 @@ jobs:
         sleep 50
         docker ps
 
+    - name: Add models
+      run: |
+        docker compose exec --workdir /pecan/scripts executor sh add.models.sh
```

Comment:
Latest logs are showing that the executor image doesn't have psql installed.

There are probably better ways, but I think this should work:

```
docker compose cp scripts/add.util.sh postgres:/tmp/add.util.sh
docker compose cp scripts/add.models.sh postgres:/tmp/add.models.sh
docker compose exec postgres /tmp/add.models.sh
```

Reply:
Starting to suspect I'm sending you on a goose chase here. Hey @robkooper is this the right approach or should we actually be, like, starting the monitor service and waiting for it to register sipnet for us?